doc-content/enterprise-only/openshift/environment-immutable-monitoring-proc.adoc

```diff
@@ -17,7 +17,7 @@ To deploy {LOCAL_CENTRAL} and Smart Router for an environment with immutable ser
 +
 [subs="attributes,verbatim,macros"]
 ----
-oc new-app -f <template-path>/{PRODUCT_INIT}{ENTERPRISE_VERSION_SHORT}-prod-immutable-monitor.yaml -p {CENTRAL_CAPITAL_UNDER}_HTTPS_SECRET={CENTRAL_ONEWORD}-app-secret
+oc new-app -f <template-path>/{PRODUCT_INIT}{ENTERPRISE_VERSION_SHORT}-prod-immutable-monitor.yaml -p {CENTRAL_CAPITAL_UNDER}_HTTPS_SECRET={CENTRAL_ONEWORD}-app-secret -p KIE_SERVER_ROUTER_HTTPS_SECRET=smartrouter-app-secret
```

Comment:
Missing closing underscore to complete the italics, so everything after it in the rest of the file is ital.